pages/national_resources/cz_resources.md

```diff
@@ -0,0 +1,108 @@
+---
+
+title: Czech Republic
+country_code: CZ
+contributors: [Karel Berka]
+coordinators: [Karel Berka]
+
+# Link to other pages in the tool assembly section on the RDMkit by listing the page_id 
+related_pages:
+  tool_assembly: [ ]
+
+training:
+  - name: Training in TeSS
+    registry: TeSS
+    url: https://tess.elixir-europe.org/content_providers/elixir-the-czech-republic-node
+  - name: ELIXIR CZ YouTube
+    registry: Youtube
+    url: https://www.youtube.com/channel/UCt0SKet24szBGjN-V1d4EKw
+  - name: ELIXIR-CZ Github
+    registry: Github
+    url: https://github.com/ELIXIR-CZ
+ - name: ELIXIR CZ Zenodo
+    registry: Zenodo
+    url: https://zenodo.org/communities/elixir-cz/?page=1&size=20
+
+ref_to_main_resources: 
+  -  Galaxy
+
+national_resources: 
+https://du.cesnet.cz/cs/navody/narodni_repozitar/start
+  - name: Galaxy Metacentrum
+    description: Galaxy Metacentrum is a Galaxy instance managed by the Czech ELIXIR node and e-Infra. It provides extra support for [RepeatExplorer](https://repeatexplorer-elixir.cerit-sc.cz/) tool for plant genomic analysis
+    how_to_access: 
+    instance_of: Galaxy
+    related_pages:
+      tool_assembly: [ ]
+      your_domain: [plant]
+      your_role: [researcher]
+      your_tasks: [data analysis]
+    url: https://galaxy.metacentrum.cz/
+    registry:
+      biotools: repeat_explorer
+     - name: e-Infra Supercomputing and data services 
+    description: e-Infra is the Czech Republic integrated high-performance research computing/data storage environment, providing world-class services to government, industry, and researchers. It also cooperates with European Open Science Cloud (EOSC) implementation in the Czech Republic
+    how_to_access:
+    related_pages: 
+      tool_assembly: []
+      your_domain: []
+      your_role: [data manager, IT support]
+      your_tasks: [data analysis, storage]
+    url: https://www.e-infra.cz/en
+- name: Data Stewardship Wizard (DSW)
+    description: DSW is a tool to create, plan, collaborate, and bring your data management plans to life.
+    how_to_access:
+    related_pages: 
+      tool_assembly: []
+      your_domain: []
+      your_role: [data manager, researcher]
+      your_tasks: [DMP]
+    url: https://ds-wizard.org/
+- name: Owncloud
+    description: Owncloud is a 100 GB cloud storage freely available for Czech scientists 
+    how_to_access: 
```

Comment:
I think it would be helpful to provide here information (gist of) identity providers (point 2 mentioned at https://du.cesnet.cz/en/navody/owncloud/start)